lib/rules/alpha-value-notation/index.js

```diff
@@ -23,27 +23,11 @@ const meta = {
 	fixable: true,
 };
 
-const ALPHA_PROPS = new Set([
-	'opacity',
-	'shape-image-threshold',
-	// SVG properties
-	'fill-opacity',
-	'flood-opacity',
-	'stop-opacity',
-	'stroke-opacity',
-]);
-const ALPHA_FUNCS = new Set([
-	'color',
-	'hsl',
-	'hsla',
-	'hwb',
-	'lab',
-	'lch',
-	'oklab',
-	'oklch',
-	'rgb',
-	'rgba',
-]);
+const ALPHA_PROPS_REGEXP =
+	/^(?:opacity|shape-image-threshold|fill-opacity|flood-opacity|stop-opacity|stroke-opacity)$/i;
+const ALPHA_FUNCTION_REGEXP = /(?:color|hsla?|rgba?|hwb|lab|lch|oklab|oklch)\(/i;
```

Comment:
question

what's the `\(` there for ?

Reply:
Something like `lab` could be part of a larger word.
However `lab(` is much less likely, `\(` in this regexp matches the opening parenthesis.

_It could still have a prefix that makes it something other than the `lab()` function._

example :

```css
.foo {
	/* oklab keyword doesn't match /oklab\(/ */
	background-image: linear-gradient(in oklab 90deg, red 25% 50%, blue 70% 75%);
}

.foo {
	/* oklab function matches /oklab\(/ */
	background-image: oklab(100% 50% 50%);
}
```